src/api/config/routes.rb

```diff
@@ -210,6 +210,9 @@ def self.public_or_about_path?(request)
 
         resource :log, only: :show, controller: 'webui/cloud/upload_job/logs'
       end
+      resource :azure do
```

Comment:
IIRC if you don't use the `` only: []`` it will create routes for azure as well.

Reply:
`only: []` added.